paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/PaymentOptionsStateMapper.kt

```diff
@@ -38,12 +38,11 @@ internal class PaymentOptionsStateMapper(
 
     @Suppress("ReturnCount")
     private fun createPaymentOptionsState(
-        paymentMethods: List<PaymentMethod>?,
+        paymentMethods: List<PaymentMethod>,
         currentSelection: PaymentSelection?,
         isLinkEnabled: Boolean?,
         googlePayState: GooglePayState,
     ): PaymentOptionsState? {
-        if (paymentMethods == null) return null
         if (isLinkEnabled == null) return null
```

Comment:
Should we do the same for Link?

Reply:
isLinkEnabled should never be null for flow controller/payment options (in practice).

I think it's fine to leave as is, or as an incident remediation.